tests/unit/python/foglamp/common/test_configuration_manager.py

```diff
@@ -2192,3 +2192,78 @@ def mock_coro():
+        read_val = {'allowPing': {'default': 'true', 'description': 'Allow access to ping', 'value': 'true', 'type': 'boolean'},
+                    'enableHttp': {'default': 'true', 'description': 'Enable HTTP', 'value': 'false', 'type': 'boolean'}}
+        payload = {'updates': [{'json_properties': [{'path': ['enableHttp', 'value'], 'column': 'value', 'value': 'false'}],
+                                'return': ['key', 'description', {'format': 'YYYY-MM-DD HH24:MI:SS.MS', 'column': 'ts'}, 'value'],
+                                'where': {'value': 'rest_api', 'column': 'key', 'condition': '='}}]}
+        audit_details = {'items': {'enableHttp': {'oldValue': 'true', 'newValue': 'false'}}, 'category': category_name}
+        storage_client_mock = MagicMock(spec=StorageClientAsync)
+        c_mgr = ConfigurationManager(storage_client_mock)
+        with patch.object(c_mgr, 'get_category_all_items', return_value=async_mock(cat_info)) as patch_get_all_items:
+            with patch.object(c_mgr._storage, 'update_tbl', return_value=async_mock(update_result)) as patch_update:
+                with patch.object(c_mgr, '_read_category_val', return_value=async_mock(read_val)) as patch_read_val:
+                    with patch.object(AuditLogger, '__init__', return_value=None):
+                        with patch.object(AuditLogger, 'information', return_value=async_mock(None)) as patch_audit:
+                            with patch.object(ConfigurationManager, '_run_callbacks', return_value=async_mock(None)) \
+                                    as patch_callback:
+                                await c_mgr.update_configuration_item_bulk(category_name, config_item_list)
+                            patch_callback.assert_called_once_with(category_name)
+                        patch_audit.assert_called_once_with('CONCH', audit_details)
+                patch_read_val.assert_called_once_with(category_name)
+            args, kwargs = patch_update.call_args
+            assert 'configuration' == args[0]
+            assert payload == json.loads(args[1])
+        patch_get_all_items.assert_called_once_with(category_name)
+
+    async def test_update_configuration_item_bulk_no_change(self, category_name='rest_api'):
+        async def async_mock(return_value):
+            return return_value
+
+        cat_info = {'enableHttp': {'default': 'true', 'description': 'Enable HTTP', 'type': 'boolean', 'value': 'true'}}
+        config_item_list = {"enableHttp": "true"}
+        storage_client_mock = MagicMock(spec=StorageClientAsync)
+        c_mgr = ConfigurationManager(storage_client_mock)
+        with patch.object(c_mgr, 'get_category_all_items', return_value=async_mock(cat_info)) as patch_get_all_items:
+            result = await c_mgr.update_configuration_item_bulk(category_name, config_item_list)
+            assert result is None
+        patch_get_all_items.assert_called_once_with(category_name)
```

Comment:
should not we add `patch_update` not called (and may be also not the patch_callback, patch_audit, patch_read_val)